core/src/main/java/com/rudderstack/android/sdk/core/Constants.java

```diff
@@ -26,6 +26,8 @@
     static final long REPEAT_INTERVAL = 1;
     // the TimeUnit in which the repeatInterval is specified. It can be either minutes / hours.
     static final TimeUnit REPEAT_INTERVAL_TIME_UNIT = TimeUnit.HOURS;
+    // whether we should automatically collect advertisement Id.
+    static final boolean AUTO_COLLECT_ADVERT_ID = true;
```

Comment:
I think default set to be `false`.

Reply:
Okay, I had set the default value to true, so that we will have the same behavior and on top of that we are giving the user flexibility to disable that.

Reply:
Shall I keep it to be true (or) change it to false ?

Reply:
AdvertisingId is not mandatory. We should make it false.

Reply:
Okay Sure